docs/graphql-admin/locales.mdx

```diff
@@ -0,0 +1,261 @@
+# GraphQL Admin: Locale overview
+
+This overview explains how to manage locales using the [GraphQL Admin API](/docs/graphql-admin).
+
+## Introduction
+
+The BigCommerce GraphQL Admin API enables developers to manage language settings across different channels. By accessing locale data and related configurations, developers can more easily localize headless storefronts. 
+
+<Callout type="info">
+Locale management is available on Catalyst storefronts.
+</Callout>
+
+## Getting started
+
+With the GraphQL Admin API, merchants can add, update, or remove locales for each sales channel, change locale status, and set a locale as the default. Below are examples of GraphQL queries and mutations for retrieving and managing locale settings.
+
+## Example requests
+This section provides examples of GraphQL Admin API queries and mutations that demonstrate how to manage locales, along with the required HTTP configuration.
+
+```http filename="GraphQL Admin API HTTP configuration" showLineNumbers copy
+POST https://api.bigcommerce.com/accounts/{{store_hash}}/graphql
+X-Auth-Token: {{access_token}}
+Accept: application/json
+Content-Type: application/json
+```
+
+### Query locales supported by the platform
+
+This query returns a static list of all locales currently supported by the platform.
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```graphql filename="Example query: Get locales supported by the platform" showLineNumbers copy
+    query {
+      i18n {
+        supportedLocales {
+          code
+          englishName
+          hasBuiltInTranslations
+          }
+        }
+      }
+    ```
+  </Tab>
+<Tab>
+```json filename="Example query: Get locales supported by the platform" showLineNumbers copy
+     {
+  "data": {
+    "i18n": {
+      "supportedLocales": [
+        {
+          "code": "ua",
+          "englishName": "Ukrainian",
+          "hasBuiltInTranslations": "false"
+        },
+        {
+          "code": "en-US",
+          "englishName": "English (United States)",
+          "hasBuiltInTranslations": "true"
+        },
+        {
+          "code": "en-UK",
+          "englishName": "English (United Kingdom)",
+          "hasBuiltInTranslations": "true"
+        },
+        {
+          "code": "fr",
+          "englishName": "French",
+          "hasBuiltInTranslations": "true"
+        }
+      }
+    }
+  }
+```
+</Tab>
+</Tabs>
+
+### Query channel locales
```

Comment:
@bc-traciporter this query is needed too. It allows to query locales enabled on a channel